packages/web-components/src/components/cbp-toast/cbp-toast.stories.tsx

```diff
@@ -22,11 +22,12 @@ export default {
     },
   };
   
-  const Template = ({ icon, content, buttons, timer, color, context, sx }) => {
+  const Template = ({ open, icon, content, buttons, duration, color, context, sx }) => {
     return ` 
           <cbp-toast
+            ${open ? `open=${open}` : ''}
```

Comment:
yout don't need the  "=${open}" here. Just the "open" attribute being present.

Reply:
updated as requested